model_info/ptwiki.draft_quality.md

```diff
@@ -1,7 +1,7 @@
 Model Information:
 	 - type: GradientBoosting
 	 - version: 0.2.1
-	 - params: {'scale': False, 'population_rates': None, 'subsample': 1.0, 'loss': 'deviance', 'max_depth': 7, 'max_features': 'log2', 'min_impurity_decrease': 0.0, 'validation_fraction': 0.1, 'n_estimators': 700, 'min_impurity_split': None, 'criterion': 'friedman_mse', 'labels': ['OK', 'spam', 'unsuitable'], 'min_samples_split': 2, 'min_samples_leaf': 1, 'init': None, 'center': False, 'multilabel': False, 'max_leaf_nodes': None, 'learning_rate': 0.01, 'label_weights': None, 'n_iter_no_change': None, 'presort': 'auto', 'verbose': 0, 'random_state': None, 'tol': 0.0001, 'min_weight_fraction_leaf': 0.0, 'warm_start': False}
+	 - params: {'loss': 'deviance', 'verbose': 0, 'n_estimators': 700, 'presort': 'auto', 'multilabel': False, 'min_samples_leaf': 1, 'min_impurity_split': None, 'max_depth': 7, 'random_state': None, 'n_iter_no_change': None, 'labels': ['OK', 'spam', 'unsuitable'], 'validation_fraction': 0.1, 'init': None, 'scale': False, 'warm_start': False, 'tol': 0.0001, 'max_features': 'log2', 'center': False, 'min_samples_split': 2, 'min_weight_fraction_leaf': 0.0, 'min_impurity_decrease': 0.0, 'population_rates': None, 'max_leaf_nodes': None, 'subsample': 1.0, 'learning_rate': 0.01, 'label_weights': None, 'criterion': 'friedman_mse'}
```

Comment:
Looks like the n_estimators here don't match what I see in the makefile lines above.  I think you need to rebuild the model using those parameters.

Reply:
Rebuilt model with new parameters.